.github/workflows/csi-test.yml

```diff
@@ -0,0 +1,63 @@
+name: Test custom storage initializer
+on:
+  pull_request:
+    paths:
+      - "csi/**"
+      - "internal/server/openapi/api_model_registry_service*"
+      - "pkg/openapi/**"
```

Comment:
Since the job takes more or less 6mins, I am proposing to run it if and only if the `csi/` directory or the model-registry api/server has changed

Reply:
Can you kindly add it on "push" ? This way it will be continously integrated on main merge, where we have less "urgency constraints". Wdyt?

Reply:
I agree!